keps/262-ConfigurableFailurePolicy/kep.yaml

```diff
@@ -0,0 +1,29 @@
+title: Configurable Failure Policy
+kep-number: 262
+authors:
+  - "@danielvegamyhre"
+status: provisional
+creation-date: 2023-08-01
+reviewers:
+  - "@kannon92"
+  - "@vsoch"
+  - "@mimowo"
+approvers:
+  - "@ahg"
+
+see-also:
+  - "NA"
+replaces:
+  - "NA"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
lets remove this and the milestone section at the end, we don't have feature gates, so this will be confusing to the reader.

Reply:
Removed